src/test/test_corpc_version.c

```diff
@@ -606,8 +601,8 @@ test_init(void)
 				    corpc_ver_mismatch_hdlr,
 				    &corpc_ver_mismatch_ops);
 	D_ASSERTF(rc == 0, "crt_rpc_srv_register() failed, rc: %d\n", rc);
-	rc = crt_rpc_srv_register(TEST_OPC_SHUTDOWN, 0, NULL,
-				  test_shutdown_hdlr);
+	rc = crt_rpc_srv_register(TEST_OPC_SHUTDOWN, CRT_RPC_FEAT_NO_REPLY,
+				  NULL, test_shutdown_hdlr);
```

Comment:
Again, doesn't seem related.

Reply:
not related, I was fixing another problem, modified the wrong branch.